var/spack/repos/builtin/packages/libdc1394/package.py

```diff
@@ -10,17 +10,30 @@ class Libdc1394(AutotoolsPackage):
     """Library providing an API for IEEE 1394 cameras."""
 
     homepage = "https://damien.douxchamps.net/ieee1394/libdc1394/"
-    url      = "https://downloads.sourceforge.net/project/libdc1394/libdc1394-2/2.2.6/libdc1394-2.2.6.tar.gz"
+    url = "https://downloads.sourceforge.net/project/libdc1394/libdc1394-2/2.2.6/libdc1394-2.2.6.tar.gz"
 
-    maintainers = ['traversaro']
+    maintainers = ["traversaro"]
 
-    version('2.2.6', sha256='2b905fc9aa4eec6bdcf6a2ae5f5ba021232739f5be047dec8fe8dd6049c10fed')
+    version(
+        "2.2.6",
+        sha256="2b905fc9aa4eec6bdcf6a2ae5f5ba021232739f5be047dec8fe8dd6049c10fed",
+    )
 
-    depends_on('libusb')
+    depends_on("pkgconfig", type="build")
+    depends_on("libusb")
+    depends_on("sdl2")
+    depends_on("libraw1394")
 
     def configure_args(self):
         args = []
-        args.append('--disable-dependency-tracking')
-        args.append('--disable-examples')
-        args.append('--disable-sdltest')
+        args.extend(
+            [
+                "--disable-dependency-tracking",
+                "--disable-examples",
+                "--disable-sdltest",
+            ]
+        )
         return args
```

Comment:
You could simplify this by saying:
```python
return [
    ...
]
```

Reply:
Good to know. I always assumed that it was standard format to have it explicit.